Maven build

matthiaskoenig opened the pull request:
I added some changes to get the maven build working (also minor bugfix for the PlotMultiTable.

matthiaskoenig committed:
working mvn version

matthiaskoenig committed:
added contributor

draeger commented:
Looks all fine to me! Just one question: Do we really need the IntellyJ-specific modules file `sbscl.iml` in the project?

matthiaskoenig commented:
The idea files were ignored as part of this commit (i.e. removed them from repo and added to .gitignore). @shalinshah1993 You should merge things as soon as possible to avoid duplicate work. 
I fixed the MultiTable issues and fixed the junit dependencies here. But by not merging you did the work again.

shalinshah1993 committed:
Merge branch 'working-branch' into maven-build
